client/src/components/ScatterChart.stories.tsx

```diff
@@ -0,0 +1,127 @@
+import { BLUE, GREEN, GREY, PURPLE } from 'components/charts/colours';
+import React from 'react';
+import ScatterChart, { AxisProps, DataProps } from './ScatterChart';
+
+export default {
+    component: ScatterChart,
+    title: 'ScatterChart'
+};
+
+interface ChartProps {
+    title: string,
+    xAxis: AxisProps,
+    yAxis: AxisProps,
+    dataColour: string,
+    maxColour: string
+}
+
+interface DataArgs {
+    data: DataProps[],
+    max: number
+}
+
+type Template = ((a: DataArgs) => JSX.Element) & { args: DataArgs };
+
+const createTemplate = (chartArgs: ChartProps): Template =>
+    Object.assign(
+        // Template function
+        (a: DataArgs) => <ScatterChart {...{ ...chartArgs, ...a }} />,
+
+        // This lets us use the args prop later on
+        { args: {} as DataArgs }
+    );
+
+// This is used to create templates for each of the scatter based charts
+const Templates = {
+    VelocityDistance: createTemplate({
+        title: "Velocity-Distance",
+        xAxis: { label: 'Distance', unit: 'km' },
```

Comment:
We tend to measure distance in meters km in our other code so far. I don't think there's a strong reason for or against either, so probably for consistency it would be good to keep to meters here

Reply:
Done
56d8bb2